samples-azure-functions/src/test/java/com/functions/EndToEndTests.java

```diff
@@ -14,6 +14,8 @@
 public class EndToEndTests {
     private static final String hostHealthPingPath = "/admin/host/ping";
     private static final String startOrchestrationPath = "/api/StartOrchestration";
+    private static final String approvalWorkFlow = "/api/ApprovalWorkflowOrchestration";
+    private static final String rewindInstance = "/api/RewindInstance";

```

Comment:
Same comment about variable naming:
```suggestion
    private static final String rewindInstanceFunctionUrl = "/api/RewindInstance";

```